browser/src/Font.ts

```diff
@@ -24,8 +24,8 @@ export function measureFont(fontFamily: string, fontSize: string, characterToTes
     document.body.appendChild(div)
 
     const rect = div.getBoundingClientRect()
-    const width = rect.width
-    const height = rect.height
+    const width = Math.ceil(rect.width)
```

Comment:
Unfortunately this `width` change is a bit problematic. The issue is that when we render whole tokens, the spacing between the letters isn't taken into account. You can see how the cursor is slightly off, and it gets worse letter-by-letter, until it's completely off by the end of the word:
![width-distortion](https://user-images.githubusercontent.com/13532591/34549686-6aedf504-f0bf-11e7-9a27-df4a850ff01e.gif)

Reply:
I tried a fix along these lines, and was able to address this particular issue by adding a 'kerning' value to add space between all the letters - using the difference between `Math.ceil(rect.width)` and `rect.width`. However, it actually looked really bad (unfortunately), so although it did address the cracks / subpixel rendering, it looked pretty bad.